xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/BitstampAuthenticatedV2.java

```diff
@@ -655,6 +656,16 @@ WithdrawalRequest[] getWithdrawalRequests(
       @FormParam("timedelta") Long timeDelta)
       throws BitstampException, IOException;
 
+  @POST
+  @Path("fees/withdrawal/")
+  WithdrawalFee[] getWithdrawalFees(
```

Comment:
can you use List here instead of array?

Reply:
As far as I know there is no performance differences there. I personally see the replacement with `List` as a part of code update activities, but again it is my humble opinion :smile: 

Let's ask @timmolter for his opinion.

I've also created small test for it and replaced the return type with list. Successfully tested with integration test, so it is working. See https://github.com/knowm/XChange/pull/4907

Reply:
Happy to `List`-ify this new endpoint as well as `getWithdrawalRequests()` since both are already being translated into Lists inside `BitstampAuthenticatedV2`. The `getUserTransactions()` endpoint is a little trickier, since that returns `BitstampUserTransaction[]` all over the place, in an outward-facing way that even my own application is currently using.

Reply:
Made the `List` change for `getWithdrawalFees()` and `getWithdrawalRequests()`